README.md

```diff
@@ -39,6 +44,38 @@ Currently, the codebase only implements n-out-of-n sharing. While t-out-of-n sha
 
 Additionally, no notions of Identifiable Aborts are implemented. If a node crashes, the protocol will halt until that node comes back online. In addition to implementing the necessary cryptographic checks to identify and attribute malicious behavior, some notion of synchronous timeouts is also required.
 
-Furthermore, the Key Refreshing portion of Auxilary Info & Key Refresh (CGGMP20 Figure 6) is not yet implemented.
+Furthermore, the Key Refreshing portion of Auxiliary Info & Key Refresh (CGGMP20 Figure 6) is not yet implemented.
+
+While some thought has been put into handling invalid messages (duplicate messages are ignored, as are some malformed ones), this has not been evaluated fully. Additionally, message authenticity (i.e. that a given message is actually coming from the sender in the "sender" field) is currently assumed to be handled outside of the protocol, by whatever networking code is shuttling messages around.
+
+
+## How to Build and Run
+
+You should try to build and run using the stable version of Rust instead of nightly. You can switch using the following:
+
+`rustup default stable` 
+`rustup update`
```

Comment:
nit: The first command here does what you said (switch to stable); the second updates the version. I think it's still useful to have both, but consider updating the text above to be clear about that, e.g.
> The library requires a recent, stable version of Rust. You can switch to stable releases and update to the latest version using the following: